config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -365,36 +365,36 @@ periodics:
     preset-service-account: "true"
   spec:
     containers:
-    - args:
-      - --check-leaked-resources
-      - --cluster=
-      - --extract=ci/latest-1.20
-      - --gcp-zone=us-west1-b
-      - --ginkgo-parallel=8
-      - --provider=gce
-      - --gcp-nodes=2
-      - --test=false
-      - --test-cmd=$GOPATH/src/sigs.k8s.io/windows-testing/gce/run-e2e.sh
-      - --test_args=--ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]
-        --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Feature:\s?[^(Windows)]\w+\]|\[Slow\]|(GMSA\ssupport\sworks\send\sto\send) --minStartupPods=8
-      - --test-cmd-args=--ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]
-        --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Feature:\s?[^(Windows)]\w+\]|\[Slow\]|(GMSA\ssupport\sworks\send\sto\send) --minStartupPods=8
-      - --test-cmd-args=--node-os-distro=windows
-      - --timeout=230m
-      command:
-      - runner.sh
-      - /workspace/scenarios/kubernetes_e2e.py
-      env:
-      - name: WINDOWS_NODE_OS_DISTRIBUTION
-        value: win2019
-      - name: PREPULL_YAML
-        value: prepull-head.yaml
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20210403-e49d2c6-1.20
-      name: ""
-      resources: {}
+      - args:
+          - --check-leaked-resources
+          - --cluster=
+          - --extract=ci/latest-1.21
+          - --gcp-zone=us-west1-b
+          - --ginkgo-parallel=8
+          - --provider=gce
+          - --gcp-nodes=2
+          - --test=false
+          - --test-cmd=$GOPATH/src/sigs.k8s.io/windows-testing/gce/run-e2e.sh
+          - --test_args=--ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]
+            --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Feature:\s?[^(Windows)]\w+\]|\[Slow\]|(GMSA\ssupport\sworks\send\sto\send) --minStartupPods=8
+          - --test-cmd-args=--ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]
+            --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Feature:\s?[^(Windows)]\w+\]|\[Slow\]|(GMSA\ssupport\sworks\send\sto\send) --minStartupPods=8
+          - --test-cmd-args=--node-os-distro=windows
+          - --timeout=230m
+        command:
+          - runner.sh
+          - /workspace/scenarios/kubernetes_e2e.py
+        env:
+          - name: WINDOWS_NODE_OS_DISTRIBUTION
+            value: win2019
+          - name: PREPULL_YAML
+            value: prepull-head.yaml
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20210403-e49d2c6-1.20
```

Comment:
does this image need to be updated or does that happen automatically?

Reply:
fixed. thanks @jsturtevant !

Reply:
lgtm! Thanks @dims !